ui/src/pages/session-details/playback/capture-details/capture-job/process-now.tsx

```diff
@@ -18,7 +20,19 @@ export const ProcessNow = ({
   const { project } = useProjectDetails(projectId as string, true)
   const { createJob, isLoading, isSuccess } = useCreateJob()
   const icon = isSuccess ? IconType.RadixCheck : undefined
-  const disabled = !capture || capture.hasJobInProgress || !pipelineId
+  const disabled = !capture || capture.hasJobInProgress || !pipelineId 
+
+  const getPipelineInfo = async () => {
```

Comment:
We might consider calling the pipeline /info endpoint from the frontend in the future, but let's just call it from the backend for now.